src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTask.java

```diff
@@ -122,11 +127,14 @@ private void leaseAndProcessZipFile(
 
         leaseAcquirer.ifAcquiredOrElse(
             blobClient,
-            leaseId -> processZipFile(container, blobClient, zipFilename, leaseId),
+            leaseId -> {
+                if (ocrValidationRetryManager.isReadyToRetry(blobClient)) {
```

Comment:
Method name seems a bit weird, are we retrying here or is it the first attempt?
If it doesn't matter then perhaps this should be renamed?

Reply:
Renamed to `canProcess`